### PR TITLE
Bug 1806974: Kubevirt vm real node name

### PR DESCRIPTION
**Pull request by yaacov**
The page displays the migration target node, instead the source node the VM was originally running on. VMI contains correct Node (the original one). Interestingly, after cca 5 or 6 minutes, the VM Status is suddenly 'Stopping', without invoking any action. VMI again reads correct phase Running.

Description:
_Problem A_ - we choose the VM pod by the last luncher pod created, in case of failed migration this will be the failed migration pod.
_Fix A_ - try to choose ready pods.

_Problem B_ - we show the VM node by checking the pod node and not the VMI node.
_FIx B_ - show node using VMI.

Screenshoot:
After:
Peek 2020-02-25 13-10

Before:
Peek 2020-02-25 13-17

### CODE REVIEW

**Comment by yaacov** [comment]
@irosenzw @glekner @suomiy please review

File: frontend/packages/kubevirt-plugin/src/selectors/pod/selectors.ts [hunk 1]
```diff
@@ -47,26 +46,29 @@ export const isPodSchedulable = (pod: PodKind) => {
   );
 };
 
+const isPodReady = (pod: PodKind): boolean =>
+  pod?.status?.phase === 'Running' && pod?.status?.containerStatuses?.every((s) => s.ready);
+
 export const findVMPod = (
```
Comment (irosenzw) [inline frontend/packages/kubevirt-plugin/src/selectors/pod/selectors.ts]: why not changing the name to `findVMIPod` too ?

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/selectors/pod/selectors.ts]: Thansk ! fixing

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/selectors/pod/selectors.ts]: done :+1:

**Pull request merged**